### DISCUSSION TITLE
Styling the body element

### DISCUSSION THREAD

**Discussion by bistace** [DioxusLabs/dioxus#616] [Q&A] [answered]
Hello,

I'm new to Dioxus so it's probably a silly question but I searched in the docs and in the book and could not find an answer. Basically, I would like to create a div that fills the entire screen so I would need to have something like this on the body element: 
```css
margin: 0
padding: 0
```

Doing this did not work, could you please show me the way to do it?
```rust
rsx! {
    body {
        margin: "0px",
        padding: "0px",
        background_color: "blue",
    }
}
```

**Comment by saicu** [comment, marked as answer]
If you're using the dioxus-cli, you can just add a CSS file to the config, see this example: https://github.com/DioxusLabs/docsite/blob/master/Dioxus.toml#L31
E.g `style = ["/custom.css"]`)

Then you can style the body in that file.
public/custom.css
```
body, #main {
        margin: "0px",
        padding: "0px",
        background-color: "blue",
}

```

Just make sure its accessible through the static asset dir, configured here: https://github.com/DioxusLabs/docsite/blob/master/Dioxus.toml#L15

Also, you may need to style the `<div id="main"></div>` as well, removing its margin/padding

**Reply by bistace** [reply]
Oh, thank you! It was indeed very simple!

**Comment by pcgeek86** [comment]
I am creating a desktop application. I tried configuring `Dioxus.toml` and creating the `style.css` file in my `public` asset directory, but for some reason, the app won't pick it up.

I also had the `style` set to `["/style.css"]` but was trying different options.

<img width="894" alt="image" src="https://github.com/DioxusLabs/dioxus/assets/466713/a80a698b-a500-4ca7-a1e8-da09c14a9379">

<img width="508" alt="image" src="https://github.com/DioxusLabs/dioxus/assets/466713/1abf0dbd-d8f1-489d-9796-35ac026962a9">
